cfgv.py

```diff
@@ -162,6 +162,12 @@ def _no_additional_keys_check(self, dct):
         )
 
 
+def _warn_additional_keys_check(self, dct):
+    extra = sorted(set(dct) - set(self.keys))
+    if extra:
+        self.callback()
```

Comment:
callback should be passed `extra` and `self.keys` probably

Reply:
True, the parameters are missing from callback() (we will do not know what to print :) )
Thanks for the hints about the coverage branches.